sdk/storage/azure-storage-queues/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 12.0.0-beta.4 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 12.0.0-beta.4 (2022-03-09)
 
 ### Other Changes
 
+- No public changes in this release.
+
```

Comment:
What's motivating the release then? What has actually changed? Implementation details?

Reply:
Yes, after this release every package will have its own SAS version, rather than share the same version defined in common package.